node/test-service/src/lib.rs

```diff
@@ -295,9 +298,33 @@ where
 
 		self.rpc_handlers.send_transaction(extrinsic.into()).await
 	}
+
+	/// Call `register_para` to register a parachain using the parachain id, validation code and head data given in
+	/// parameter.
+	pub async fn register_para(
+		&self,
+		para_id: ParaId,
+		binary: ValidationCode,
+		head_data: HeadData,
+	) -> Result<(), RpcTransactionError> {
+		let function = polkadot_test_runtime::Call::Sudo(pallet_sudo::Call::sudo(Box::new(
+			polkadot_test_runtime::Call::Registrar(registrar::Call::register_para(
+				para_id,
+				Info {
+					scheduling: Scheduling::Always,
+				},
+				binary,
+				head_data,
+			)),
+		)));
+
+		self.call_function(function, Sr25519Keyring::Alice).await?;
+
+		Ok(())
```

Comment:
```suggestion
		self.call_function(function, Sr25519Keyring::Alice).await
```

Reply:
The result is different. This function returns `()` because the return value is not very useful. (`call_function` returns the RPC result which could (?) be potentially useful)

Reply:
Than do a `map`

Reply:
:thinking: not convinced

Reply:
Why?

Reply:
Different people have different taste. I find the other one more readable. But yours is a bit more expliclit which is good too